runtime/src/main/java/io/quarkiverse/githubapp/runtime/Routes.java

```diff
@@ -63,6 +67,10 @@ public class Routes {
     Path tmpDirectory;
 
     public void init(@Observes StartupEvent startupEvent) throws IOException {
+        if (launchMode != LaunchMode.TEST) {
+            configChecker.checkConfig();
+        }
+
```

Comment:
Shouldn't this (and maybe the "Payload signature checking ..." log below) got to `ConfigChecker`?